### DISCUSSION TITLE
Rules to the array in the request body

### DISCUSSION THREAD

**Discussion by eddyheng** [mockoon/mockoon#631] [Help] [answered]
the request body likes this:
`
{
	"groupName": "1",
	"seq":"3",
	"ids":["139123323456","17787645635","15567354657"],
	"members":[
		{
			"name":"Andrew",
			"Country":"Japan"
		},
		{
			"name":"Tracy",
			"Country":"US"
		},
		{
			"name":"Edward",
			"Country":"Germany"
		}
	]
}
`
I want to judge 2 things in the Rules：
1、Does it contain the string begin with 177 in the ids;
2、In the array members，the second JSON object，whether the value of name is Tracy

How should I create the Rules？

**Comment by 255kb** [comment, marked as answer]
This should work: 
![image](https://user-images.githubusercontent.com/7489814/151767801-9b31c42f-672e-4cc3-a6e0-85f1478e7ff1.png)

When regex match encounter an array it will test the regex against each array item. Also, the rules support object paths, that's why you can use this: `members.1.name`.

[More info](https://mockoon.com/docs/latest/route-responses/dynamic-rules/)

**Reply by eddyheng** [reply]
got it. very thankful
